### PR TITLE
iqproj

### PR DESCRIPTION
**Pull request by joechrz**
:D

### CODE REVIEW

**Comment by joechrz** [comment]
@scamden mind giving it a quick once-over locally just to make sure everything works like you'd expect?

File: package.json [hunk 1]
```diff
@@ -17,12 +24,26 @@
   },
   "devDependencies": {
     "aliasify": "^1.5.0",
-    "gulp": "^3.8.0",
-    "gulp-tasks-riq": "^0.1.0",
+    "browserify": "~12.0.1",
+    "browserify-istanbul": "~0.2.1",
+    "chokidar-cli": "~1.1.1",
```
Comment (scamden) [inline package.json]: we need the chokidar cli?

**Comment by scamden** [comment]
lgtm just had the one question

File: package.json (hunk 1 above)
Comment (joechrz) [inline package.json]: Yep.  Used to watch the sass (since Libsass doesn't have built in watch) and the release/ folder for live reload

Comment (scamden) [inline package.json]: oh right. sweet.

**Comment by joechrz** [comment]
@scamden de-bowered!

**Pull request merged**